Template/Tag/TawkToTag.web.js

```diff
@@ -1,10 +1,12 @@
 (function () {
     return function (parameters, TagManager) {
+        var Tawk_API = Tawk_API || {}, Tawk_LoadStart = new Date();
         this.fire = function () {
             var tawkToId = parameters.get("tawkToId");
+            var tawkToWidgetId = parameters.get("tawkToWidgetId");
             var s1 = document.createElement("script"), s0 = document.getElementsByTagName("script")[0];
             s1.async = true;
-            s1.src = 'https://embed.tawk.to/' + tawkToId + '/default';
+            s1.src = 'https://embed.tawk.to/' + tawkToId + '/' + tawkToWidgetId;
```

Comment:
can we encodeURIComponent both parameters here?

Reply:
Yes added